Sources/StreamVideo/OpenApi/generated/Models/PushNotificationSettingsInput.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+    
+public final class PushNotificationSettingsInput: @unchecked Sendable, Codable, JSONEncodable, Hashable {
```

Comment:
Why is this one needed? Seems like it's the same as `PushNotificationSettingsResponse`?